mysite/urls.py

```diff
@@ -11,6 +11,5 @@
     path('api/login/', include('rest_social_auth.urls_jwt_pair')),
     path('token/', MyTokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
-    path('modify_user/', ChangeUserView.as_view()),#test
```

Comment:
Myślę, że zmianę hasła można uznać za zmianę danych usera, dlatego lepiej byłoby to zgrupować razem do accounts/. Szczególnie jak sam View też tam jest.